### PR TITLE
add potentialSourceMapExtension option

### PR DESCRIPTION
**Pull request by broksonic21**
Fixes [https://github.com/mondaychen/devtools-ignore-webpack-plugin/issues/4](https://github.com/mondaychen/devtools-ignore-webpack-plugin/issues/4) - adding a function/option to allow extension override (and sicne function, can use advanced logic)

Went with a function vs a list of extensions to match the other option

Open to alternate namings: went with `potentialSourceMapExtension` but called it out as rarely being needed

Note: I ran `npm run format` which modified the README.md in a few spots for consistency after I edited it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,40 +23,64 @@ First, follow [webpack's guide](https://webpack.js.org/configuration/devtool/) t
...
+  plugins: [new DevToolsIgnorePlugin()],
+};
 ```
 
 ## Options
 
-Currently, this plugin only supports one option: `shouldIgnorePath`. It is a function that checks whether a file should be ignored. The function takes a single argument, which is the path of the file. It should return a boolean value.
+Currently, this plugin supports two options: `shouldIgnorePath` and `potentialSourceMapExtension`.
+
+# shouldIgnorePath
```
Comment (mondaychen) [inline README.md:38]: This should be `###`. Same for the new option.

**Comment by mondaychen** [comment]
How about `isSourceMapAsset(name)`?

File: README.md (hunk 1 above)
Comment (broksonic21) [reply README.md]: updated!

**Comment by broksonic21** [comment]
> How about `isSourceMapAsset(name)`?

I like it - updated!

**Comment by mondaychen** [comment]
Love it. Thanks!

**Comment by mondaychen** [comment]
Released as 0.1.2
Please let me know if you see anything not working as expected!

**Pull request merged**